exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/MysqlDialect.kt

```diff
@@ -243,7 +243,7 @@ internal open class MysqlFunctionProvider : FunctionProvider() {
         }
 
         val isAliasSupported = when (val dialect = transaction.db.dialect) {
-            is MysqlDialect -> dialect !is MariaDBDialect && dialect.isMysql8
+            is MysqlDialect -> dialect !is MariaDBDialect && dialect.fullVersion >= "8.0.19"
```

Comment:
A string comparison is the easiest and passes all the comparison tests because of MySQL's predictable release versioning structure.

Alternatively, I could implement the comparison similar to how it's done in `isVersionCovers`, by parsing the version string and converting it to a `BigDecimal` first.
Parsing each version component and compare them individually is also an option.